project-koku/koku
# Moving processing temp directories to be on PVC mount

Addresses #1187 
Changes Made:
* Temp directories need to map to the OpenShift PVC mount.
* Removing all csv files that do not belong to the current assembly id.

**Testing**
1.  Add AWS, Azure, and OCP providers
2. Ingest OCP data via upload service
3. Inspect temp directories on listener.  Verify they are correct
4.  Re-generate OCP data and re-upload to trigger another report processing session
5. Re-inspect temp directories.  Verify new assembly_id is present, and old one is deleted
6. Trigger download to get AWS and Azure data
7. Inspect koku worker temp directory to verify files
8.  Add an AWS-local provider and test with nise

**Test Results**
[issue_1187_ut.txt](https://github.com/project-koku/koku/files/3677622/issue_1187_ut.txt)
